airflow/dags/manual_ingestion.py

```diff
@@ -0,0 +1,22 @@
+'''Manual data ingestion DAG.'''
+from util import create_bq_ingest_operator
+# Ingore the Airflow module, it is installed in both our dev and prod environments
+from airflow.models import Variable  # type: ignore
+from airflow import DAG  # type: ignore
+from airflow.utils.dates import days_ago  # type: ignore
+
+default_args = {
+    'start_date': days_ago(0),
+}
+
+manual_ingestion_dag = DAG(
+    'manual_ingestion_dag',
+    default_args=default_args,
+    description='Triggering for manual uploads.')
+
+# Manual Uplaods
+manual_uploads_payload = {'message': {'is_airflow_run': True,
+                                      'gcs_bucket': Variable.get('GCS_MANUAL_UPLOADS_BUCKET'),
+                                      'id': 'MANUAL_UPLOADS'}}
+manual_uploads_bq_operator = create_bq_ingest_operator(
+    'manual_uploads_task', manual_uploads_payload, manual_ingestion_dag)
```

Comment:
We will still need the exporter for manual uploads. Were you planning on adding this in a separate PR?

Reply:
Ah so theoretically the BQ project that the exporter should pull from should be separate. The manual ingest stuff has an internal loop because we need human eyes on it, so it should be a somewhat separate loop.

https://github.com/SatcherInstitute/health-equity-tracker/pull/109#discussion_r529041940

So in the drawing there, this is geared towards the "bottom track" of manual loads, and the "manual landing" dataset would not be something the exporter is sourcing.

Lorenzo, I think this is what this script is for, but correct me if I'm mis-interpreting.

Reply:
Yeah my plan was to do that in a separate PR. I think Brian had some idea that would make this process different from the other sources.

Reply:
Yeah, gotcha. The missing step is some sort of "yes, this dataset looks right, let's promote it to the public one" but we can do that in a separate issue/PR.